postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraQueueDAO.java

```diff
@@ -27,6 +27,7 @@
 	@Inject
 	public AuroraQueueDAO(DataSource dataSource, ObjectMapper mapper) {
 		super(dataSource, mapper);
+		loadQueues();
```

Comment:
Just load queue names right away. It helps container to avoid extra transaction in order to verify whether the queue exists in the db or not